integration/feature/init/src/MillInitSbtTests.scala

```diff
@@ -0,0 +1,230 @@
+
+object MillInitSbtTests {
+  val initCommand = ("init", "--base-module", "BaseModule", "--deps-object", "Deps", "--merge")
+  def bumpSbtTo1107(workspacePath: os.Path) =
+    // bump sbt version to resolve compatibility issues with lower sbt versions and higher JDK versions
+    os.write.over(workspacePath / "project" / "build.properties", "sbt.version = 1.10.7")
+}
+
+// relatively small libraries
+
+object MillInitLibraryExampleTests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 21 KB
+    - sbt 1.5.2
+     */
+    val url = "https://github.com/scalacenter/library-example/archive/refs/tags/v1.0.1.zip"
+
+    test - integrationTest(url) { tester =>
+      import tester.*
+
+      val initResult = eval(initCommand, stdout = os.Inherit, stderr = os.Inherit)
+      assert(initResult.isSuccess)
+
+      val compileResult = eval("compile")
+      assert(compileResult.isSuccess)
+    }
+  }
+}
+
+object MillInitScalaCsv200Tests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 34 KB
+    - originally sbt 1.10.0
+     */
+    val url = "https://github.com/tototoshi/scala-csv/archive/refs/tags/2.0.0.zip"
+
+    test - integrationTest(url) { tester =>
+      import tester.*
+
+      bumpSbtTo1107(workspacePath)
+
+      val initResult = eval(initCommand, stdout = os.Inherit, stderr = os.Inherit)
+      assert(initResult.isSuccess)
+
+      val compileResult = eval("compile")
+      // Cross builds are not supported yet.
+      assert(!compileResult.isSuccess)
+    }
+  }
+}
+
+object MillInitScalaCsv136Tests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 28 KB
+    - originally sbt 1.2.8
+     */
+    val url = "https://github.com/tototoshi/scala-csv/archive/refs/tags/1.3.6.zip"
+
+    test - integrationTest(url) { tester =>
+      import tester.*
+
+      bumpSbtTo1107(workspacePath)
+
+      val initResult = eval(initCommand, stdout = os.Inherit, stderr = os.Inherit)
+      assert(initResult.isSuccess)
+
+      val compileResult = eval("compile")
+      // It works here, but scala 2.11 with JDK 6 seems not supported when run with JDK 17 in shell.
+      assert(compileResult.isSuccess)
+    }
+  }
+}
+
+// same as the one in the unit tests
+object MillInitSbtMultiProjectExampleTests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 12 KB
+    - originally sbt 1.0.2
+     */
+    val url =
+      "https://github.com/pbassiner/sbt-multi-project-example/archive/152b31df9837115b183576b0080628b43c505389.zip"
+
+    test - integrationTest(url) { tester =>
+      import tester.*
+
+      bumpSbtTo1107(workspacePath)
+
+      val initResult = eval(initCommand, stdout = os.Inherit, stderr = os.Inherit)
+      assert(initResult.isSuccess)
+
+      val compileResult = eval("compile")
+      assert(compileResult.isSuccess)
+
+      val compileSubmodulesResult = eval("_.compile")
+      if (System.getProperty("java.version").split('.').head.toInt <= 11)
```

Comment:
Can use implement a `--jvm-id 11` flag similar to what the Gradle import supports, so the user can force this to use a working version of JDK11?